### PR TITLE
API Changelog 4.73.0

### PR DESCRIPTION
**Pull request by sagesyr**

### CODE REVIEW

File: src/content/changelog/4-73-0-2020-08-24.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - API
+---
+
+### Added
+
+### Changed
+
+### Fixed
+- A bug was fixed that would result in a 500 error when filtering domain records by a non-existent tag. This will now return an empty 200 respone. 
+- Previously, NS and CNAME records could be created for eachother, resulting in errors. If this is attempted now, the record creation will fail with a "Record conflict" message.
```
Comment (yoshizzle) [inline src/content/changelog/4-73-0-2020-08-24.md]: "each other" (two words)

**Review by yoshizzle — CHANGES_REQUESTED** [review]
I'd maybe also add this: https://bits.linode.com/LinodeAPI/apinext/pull/2839/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR33

File: src/content/changelog/4-73-0-2020-08-24.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+changelog:
+  - API
+---
+
+### Added
+- Added the `BETA List Maintenance` ([GET /account/maintenance](/api/v4/account-maintenance)) endpoint. This new beta endpoint allows you to view a collection of maintenance objects for any entity you have permissions to view. Currently, Linodes are the only entities available for viewing.
+
+### Fixed
+- A bug was fixed that would result in a 500 error when filtering domain records by a non-existent tag. This will now return a 200 response with an empty pagination result. 
+- Previously, NS and CNAME records could be created for each other, resulting in errors. If this is attempted now, the record creation will fail with a "Record conflict" message.
```
Comment (alsaihn) [inline src/content/changelog/4-73-0-2020-08-24.md]:
```suggestion
- Previously, NS and CNAME records could be created for each other, resulting in errors when the zone file gets rendered. If this is attempted now, the record creation will fail with a "Record conflict" message.
```

**Review by alsaihn — COMMENTED** [review]
Small clarification for where the NS/CNAME error previously showed up.

**Review by alsaihn — APPROVED** [review]

**Pull request merged**